src/ChaosBox/CLI.hs

```diff
@@ -265,15 +277,55 @@ runChaosBoxDirectly Opts {..} doRender = replicateM_ optRenderTimes $ do
       , gcMetadataString = optMetadataString
       }
 
-  void . renderWith surface . flip runReaderT ctx . flip runRandT stdGen $ do
-    cairo $ scale optScale optScale
-    void doRender
+  if optFileFormat == PNG || optRenderMode == Interactive then
```

Comment:
It seems like we'd want `&&` here; I don't want to mislead users by writing `png`s when they specify `Interactive` mode even though they've specified a different file format.